### PR TITLE
Ci react native

### PR DESCRIPTION
**Pull request by JssDWt**

### CODE REVIEW

File: snippets/react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Steps to compile the snippets locally
+1. Build a react native package
+  - By running the publish-all-platforms CI in the breez-sdk repository (use dummy binaries)
+2. Download the react-native-{VERSION} artifact 
+3. Put the `sdk-react-native.tgz` file in the `snippets/react-native/packages` folder
```
Comment (dangeross) [inline snippets/react-native/README.md:5]: Maybe just to clarify

```suggestion
3. Unzip the artifact and put the `sdk-react-native.tgz` file in the `snippets/react-native/packages` folder
```

File: snippets/react-native/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+4. happy coding
```
Comment (dangeross) [inline snippets/react-native/README.md:6]:
```suggestion
4. Run `yarn` install
5. Happy coding
```

File: snippets/react-native/package.json [hunk 3]
```diff
@@ -5,15 +5,24 @@
     "main": "index.ts",
     "license": "MIT",
     "scripts": {
-        "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+        "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+        "format": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
```
Comment (dangeross) [inline snippets/react-native/package.json:9]: Add compile for local testing

```suggestion
        "format": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
        "compile": "tsc --noEmit"
```

**Review by dangeross — CHANGES_REQUESTED** [review]
Looks overall good! Just some small comments to help locally running

**Comment by JssDWt** [comment]
Comments processed

**Pull request merged**